util/grpc/src/lib.rs

```diff
@@ -39,7 +39,6 @@ pub use crate::{
         BasicCredentials, TokenAuthenticator, TokenBasicCredentialsGenerator,
         TokenBasicCredentialsGeneratorError, ANONYMOUS_USER,
     },
-    autogenerated_code::*,
```

Comment:
removed because it was listed as unused.  Did this do anything special?

Reply:
it's a duplicate from line 21, so is uneeded